### PR TITLE
Scroll to top button

### PR DESCRIPTION
**Pull request by MohammadAlhallaq**
this PR fixes this [issue](https://github.com/brendt/rfc-vote/issues/126)

### CODE REVIEW

**Comment by MohammadAlhallaq** [comment]
also Added a button to scroll to the top of the page when scrolling too deep into the RFC argument component.

File: resources/views/layouts/base.blade.php [hunk 1]
```diff
@@ -87,6 +87,17 @@ class="container mx-auto px-4 mt-4 md:mt-12 max-w-[1200px] mb-8"
     {{ $slot }}
 </div>
 
+@if(request()->is('rfc/*'))
```
Comment (brendt) [inline resources/views/layouts/base.blade.php:90]: I don't think this is the way to go. The rfc view should pass a property to the base component like we already do with the page title:

```blade
@component('layouts.base', [
    'pageTitle' => $rfc->title . ' – RFC Vote',
])
```

File: resources/views/components/argument-card/card.blade.php [hunk 2]
```diff
@@ -51,7 +51,7 @@ class="{{ $isEditing?->is($argument) ? 'hover:text-green-800' : 'hover:text-blue
 
         <x-argument-card.card-footer
             :argument="$argument"
-            :user="$user"
+            :user="$argument->user"
```
Comment (brendt) [inline resources/views/components/argument-card/card.blade.php:54]: Could you remove this line? It's been fixed in another way. Having this line actually causes a bug somewhere else.

Comment (MohammadAlhallaq) [reply resources/views/components/argument-card/card.blade.php]: sure, Now it's back to the way it was.

**Pull request merged**